polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,134 @@
+# This script includes docking related evaluation metrics.
+
+from typing import Union, List
+from copy import deepcopy
+
+import numpy as np
+from rdkit.Chem.rdMolAlign import CalcRMS, GetBestRMS
+
+import datamol as dm
+
+
+## The following rmsd implementation are modified based on ppsebusters https://github.com/maabuu/posebusters/blob/main/posebusters/modules/rmsd.py
+
+
+def _call_rdkit_rmsd(mol_probe: dm.Mol, mol_ref: dm.Mol, conf_id_probe: int, conf_id_ref: int, **params):
+    try:
+        return _rmsd(mol_probe, mol_ref, conf_id_probe, conf_id_ref, **params)
+    except RuntimeError:
+        pass
+    except ValueError:
+        pass
+
+    return np.nan
+
+
+def _rmsd(
+    mol_probe: dm.Mol, mol_ref: dm.Mol, conf_id_probe: int, conf_id_ref: int, kabsch: bool = False, **params
+):
+    if kabsch is True:
+        return GetBestRMS(prbMol=mol_probe, refMol=mol_ref, prbId=conf_id_probe, refId=conf_id_ref, **params)
+    return CalcRMS(prbMol=mol_probe, refMol=mol_ref, prbId=conf_id_probe, refId=conf_id_ref, **params)
```

Comment:
**Question:** Do we need to support both options? Could we actually use [`dm.conformers.rmsd`](https://docs.datamol.io/stable/api/datamol.conformers.html#datamol.conformers._conformers.rmsd), which seems to use yet another method, i.e. `rdMolAlign.AlignMol()` ? How do these 3 differ?

Reply:
Here we want to compute the rmsd of two specific conformers in docking position.
`dm.conformers.rmsd` computes the rmsd of ALL the conformers of a molecule.  

After docking, the target was aligned with the original target from the crystal structure. The ligand was then extracted and used to compute the RMSD relative to the ligand in the reference position. 
Here, the metrics take only the ligands as input, assuming the ligand has already been aligned to the relevant position. So we don't want to use `AlignMol` to align them again.